ERCS/erc-0000.md

```diff
@@ -0,0 +1,68 @@
+---
+eip: 0000
```

Comment:
```suggestion
eip: 7743
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

Reply:
OK, done!